lib/src/dbs/node.rs

```diff
@@ -27,6 +28,16 @@ pub struct Timestamp {
 	pub value: u64,
 }
 
+impl Display for Timestamp {
+	fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+		let obj: Object = map! {
+			"value".to_string() => self.value.to_string().into(),
+		}
+		.into();
+		write!(f, "{}", obj)
```

Comment:
Is there any reason we need a string representation of timestamp as an object like this instead of just `self.value.to_string()`? In any case, is the display implementation necessary?

Reply:
Removed Display